modules/cnf-configuring-workload-hints.adoc

```diff
@@ -81,3 +57,28 @@ perPodPowerManagement: true
 |Critical and non-critical workloads
 |Allows for power management per pod.
 |===
+
+.Example
+
+This configuration is commonly used in telco RAN DU deployment. 
+
+[source,yaml]
+----
+    apiVersion: performance.openshift.io/v2
+    kind: PerformanceProfile
+    metadata:
+      name: workload-hints
+    spec:
+      ...
+      workloadHints:
+        realTime: true
+        highPowerConsumption: false
+        perPodPowerManagement: false <1>
+----
+<1> Disables some debugging and monitoring features that can affect system latency.
+
+[NOTE]
+====
+When the `realTime` workload hint flag is set to `true` in a performance profile, add the `cpu-quota.crio.io: disable` annotation to every guaranteed pod with pinned CPUs. This annotation is necessary to prevent the degradation of the process performance within the pod. If the `realTime` workload hint is not explicitly set then it defaults to `true`.
```

Comment:
```suggestion
When the `realTime` workload hint flag is set to `true` in a performance profile, add the `cpu-quota.crio.io: disable` annotation to every guaranteed pod with pinned CPUs. This annotation is necessary to prevent the degradation of the process performance within the pod. If the `realTime` workload hint is not explicitly set, it defaults to `true`.
```